src/pages/en/install/auto.md

```diff
@@ -33,18 +35,60 @@ yarn create astro
+Depending on your package manager, you may be prompted to confirm you want to install `create-astro@latest`. You will then be asked to specify a project folder (e.g. `./my-astro-site`) which will create a new directory.
+
+### Choose a Starter Template
+You will then see a short list of starter templates to choose from: 
+- `Just the basics`: A great starter template for anyone wanting to explore Astro.
+- `Blog`, `Documentation`, `Portfolio`: opinionated themes for specific use-cases.
+- `Completely empty`: A template that just includes the bare minimium to get started.
+
+Use the arrow keys (up and down) to navigate to the template you want to install, then press return (enter) to submit.
+
+> Want to check out the templates in your browser before choosing? Visit [astro.new](https://astro.new/)
+
+### Install Dependencies (optional)
+The wizard will offer to run an `install` command for you at this time, which is optional.
+
+> If you do not do so at this time, you will need to [install dependencies](/en/install/auto#2-install-dependencies) after the wizard has finished, before starting your project.
+
+### Install any Official Astro Integrations (optional)
+You will be given the option at this time to add any [additional UI frameworks](/en/core-concepts/framework-components) (React, Svelte, Vue, Solid, Preact, Lit) and then other Astro official integrations (Tailwind, Turbolinks, Partytown, Sitemap) by running `astro add --yes`
+
+To select which Astro integrations, if any, you would like to include in your project, use the arrow keys (up and down) to navigate and the space bar to toggle selected/unselected. You can select multiple items at once, or you can continue without selecting any items.
+
+
+When you are satisfied with your selection, press return (enter) to submit. 
+
+>🟢 Selected integrations will display a solid (green) circle. (All integrations are **unselected by default**.) 
+>
+>⚠️ If every item is unselected, then no integrations will be added at this time. 
 
-## 2. Install Dependencies
 
-When the `create-astro` install wizard is complete, you should see some recommended instructions on your screen to follow that will help you complete setup and start your new project. 
+These integrations, and any [Astro community integrations](https://astro.build/integrations), can also be added later by following the instructions in our [integrations guide](/en/guides/integrations-guide).
 
-The only required step remaining is to install your project's dependencies using a package manager like npm:
+
+After selecting your integrations to add, you should see the following terminal message followed by **the entire contents of the `astro.config.mjs` file** that Astro will include with your project:
+
+```bash
+Astro will make the following changes to your config file:
+```
+
+This message should show that your chosen integrations have been successfully added to your project configuration. (If not, you can always add them manually later.)
```

Comment:
Im not sure about this yin,

Reply:
Can you let me know what you're not sure about? I intended it as a "if you look here, you'll be able to tell integrations were added. If you don't see the things you think you added, you can go back and do it later" to deal with the issue that we still have people thinking they've installed integrations when they haven't. So not only is this a prompt to look for that, but also reassurance that it's fixable.

Do you think this isn't necessary at all? Do you think it's unclear? Just let me know!